src/workerd/api/node/zlib-util.c++

```diff
@@ -3,9 +3,159 @@
 //     https://opensource.org/licenses/Apache-2.0
 #include "zlib-util.h"
 
+#include <workerd/jsg/exception.h>
+
+#include <zlib.h>
+
+#include <iostream>
```

Comment:
```suggestion
#include <workerd/jsg/exception.h>
#include <zlib.h>
#include <iostream>
```

Reply:
ughhh.... can't wait for clang-format to be merged